terraso_backend/apps/graphql/schema/sites.py

```diff
@@ -54,6 +55,9 @@ class Meta:
 class SiteNode(DjangoObjectType):
     id = graphene.ID(source="pk", required=True)
     seen = graphene.Boolean(required=True)
+    soil_data = graphene.Field(
+        "apps.soil_id.graphql.soil_data.SoilDataNode", required=True, default_value=SoilData()
```

Comment:
```suggestion
        SoilDataNode, required=True, default_value=SoilData()
```

Reply:
unfortunately this causes an issue with circular imports (unrelated to the other issue with circular imports which david recently failed) that i believe can only be resolved by referencing `SoilDataNode` by string as above. if you can find a way to resolve the issue without this string reference i would be very into that, but so far this is the only way i've found that works!